src/components/MeasureResults/MeasureResultsRow.js

```diff
@@ -8,50 +8,50 @@ import React from 'react';
 function MeasureResultsRow({ measureResult, handleMeasureChange, measureColor }) {
   return (
     <Box>
-      <Divider color="black" />
-      <Grid container justifyContent="space-evenly" direction="row" alignItems="center" spacing={2} sx={{ width: '100%', p: '3px', m: '2px' }}>
+      <Divider color="#78909C" stroke={2} />
+      <Grid container justifyContent="space-evenly" align-items="center" direction="row" spacing={2} sx={{ width: '100%', p: '3px', m: '0px' }}>
```

Comment:
You will want to put these styles in a scss file to be consistent.

Reply:
`MeasureResultsTable` is doing many of the same things. You can probably `@extend` that.

Reply:
Can do. let me repush